committeeoversightapp/utils.py

```diff
@@ -4,28 +4,36 @@
 
 from django.core.exceptions import ObjectDoesNotExist
 
-from opencivicdata.legislative.models import EventDocument, EventDocumentLink, \
+from opencivicdata.legislative.models import EventDocument, \
+                                             EventDocumentLink, \
                                              EventParticipant
 from opencivicdata.core.models import Organization
+
 from .models import WitnessDetails, HearingCategory
 
+
 def get_ext(url):
-    """ Given a url string, find the file extension at the end """
-    
+    """Given a url string, find the file extension at the end."""
+
     path = urlparse(url).path
     ext = splitext(path)[1]
     return ext
 
+
 def archive_url(url):
     wayback_host = 'http://web.archive.org'
     save_url = '{0}/save/{1}'.format(wayback_host, url)
     archived = requests.get(save_url)
     try:
-        archive_url = '{0}{1}'.format(wayback_host, archived.headers['Content-Location'])
+        archive_url = '{0}{1}'.format(
+            wayback_host,
+            archived.headers['Content-Location']
+            )
```

Comment:
Nitpick, but it seems like this and other function calls/dictionary definitions that you've updated in this module have inconsistent indentation for the closing paren. Sometimes it's on a new line indented with the previous arg, and sometimes it's on the same line as the last arg in the list.

If the closing paren is on its own line, I usually match it to the opening line, e.g.:

 ```python
        archive_url = '{0}{1}'.format(
            wayback_host,
            archived.headers['Content-Location']
        )
```

Since we don't have a styleguide this is ultimately a personal preference thing though, as long as it's consistent I think it's fine.